source/extensions/filters/http/grpc_web/grpc_web_filter.cc

```diff
@@ -45,7 +45,6 @@ Http::FilterHeadersStatus GrpcWebFilter::decodeHeaders(Http::HeaderMap& headers,
     return Http::FilterHeadersStatus::Continue;
   }
   is_grpc_web_request_ = true;
-
```

Comment:
revert unrelated change

Reply:
Sorry, forgot to check before submission